### PR TITLE
Added project id and forgot password features

### PR DESCRIPTION
**Pull request by shweta2101**
1. Added project id as a part of display in :
   - Dashboard screen (for non-admin)
   - List of project groups (for admin)
2. Added forgot password button which on click display a modal.

Screenshot 2023-11-21 at 5 39 10 PM
Screenshot 2023-11-21 at 6 16 55 PM
Screenshot 2023-11-21 at 6 07 30 PM
Screenshot 2023-11-21 at 5 52 03 PM

### CODE REVIEW

File: tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                <Link className="tdei-primary-link" onClick={handleShow}>
+                    Click Here
+                </Link>
+            </div>
+            <Modal show={show} onHide={handleClose}>
+                <Modal.Header closeButton>
+                    <Modal.Title>Forgot Password?</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <p>Please send an email with your username to : admin@tdei.com</p>
```
Comment (sureshgaussian) [inline tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js:24]: Please change the email to tdei@uw.edu

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
Please change the admin email to contact at as tdei@uw.edu

File: tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js (hunk 1 above)
Comment (MashB) [inline tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js:24]: Please get the email from environment variable. Emails will change over time.

**Review by MashB — CHANGES_REQUESTED** [review]

File: tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js (hunk 1 above)
Comment (shweta2101) [reply tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js]: Changed the email : 
Screenshot 2023-11-27 at 10 42 25 AM

Comment (shweta2101) [reply tdei-ui/src/components/ForgotPassModal/ForgotPassModal.js]: Added email as an environmental variable : 
Screenshot 2023-11-27 at 10 52 50 AM

**Review by MashB — APPROVED** [review]

**Pull request merged**